go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/stretchr/testify v1.8.1
 	golang.org/x/sync v0.0.0-20220722155255-886fb9371eb4
 	google.golang.org/grpc v1.49.0
-	gopkg.in/yaml.v2 v2.4.0
+	gopkg.in/yaml.v2 v2.4.0 // indirect
```

Comment:
All the changes in go.sum and in this file are just due to a routine `go mod tidy`.